src/aurelia-slickgrid/models/aureliaViewOutput.interface.ts

```diff
@@ -1,7 +1,8 @@
-import { View, ViewSlot } from 'aurelia-framework';
+import { ICustomElementController } from '@aurelia/runtime-html';
 
 export interface AureliaViewOutput {
-  bindings?: any;
-  view?: View;
-  viewSlot?: ViewSlot;
+  // bindings?: any;
+  // view?: View;
+  // viewSlot?: ViewSlot;
+  controller?: ICustomElementController<any>;
```

Comment:
so all these 3 props got replaced by 1 new `controller` prop and that's it? I will have to keep that in mind for the migration since that will be a breaking change for sure. I also assume you left the comments for me to see the differences, you could go ahead and delete them now that I've seen it :)

Reply:
I assumed that these were only needed to properly dispose of the dynamic content, so not sure about anyone relying on them